app/src/main/res/layout/fragment_crime.xml

```diff
@@ -6,17 +6,52 @@
     android:layout_height="match_parent"
     android:layout_margin="16dp">
 
-    <TextView
-        style="?android:listSeparatorTextViewStyle"
+    <LinearLayout
```

Comment:
Why not use a `ConstraintLayout`? You wouldnt then need to nest multiple `LinearLayout`s inside

Reply:
Changed to `ConstraintLayout`
Looks much clenaer now